example/example.rb

```
@@ -82,4 +82,4 @@
 }
 ctx2.track("payment", properties)
 
-ctx2.close
+ctx2.close
```

RuboCop (1.82.1)

[convention] Final newline missing.

(Layout/TrailingEmptyLines)